Gemfile

```diff
@@ -49,5 +93,10 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+group :production do
+  # Use postgresql as the database for Active Record
+  gem 'pg', '~> 0.21.0'
```

Comment:
why do you just use 'pg' on production, how about test and development environment?